Minigin/Font.h

```diff
@@ -20,6 +20,6 @@ namespace dae
 		Font & operator= (const Font &&) = delete;
 	private:
 		_TTF_Font* m_font;
-		unsigned int m_size;
+		[[maybe_unused]] unsigned int m_size;
```

Comment:
(std::remove(

Reply:
Fair enough. I figured the field might be there for convenience, but it's better to not have it in the first place unless needed. I've now removed it.